app/components/Sidebar/components/CollectionLink.tsx

```diff
@@ -264,17 +275,17 @@ function CollectionLink({
             />
           </DropToImport>
         </Draggable>
-        {expanded && manualSort && (
+        {displayDocumentLinks && manualSort && (
           <DropCursor isActiveDrop={isOverReorder} innerRef={dropToReorder} />
         )}
-        {isDraggingAnyCollection && (
+        {!displayDocumentLinks && (
```

Comment:
I haven't tried it yet but on first glance, this seems strange to me - are you sure this is right? This appears to show drop cursor when there is no active drag happening. Should this just remain `isDraggingAnyCollection`?

Reply:
Yeah, you are right.